views/js/qtiCreator/tpl/inlineInteraction/inlineChoiceInteraction.tpl

```diff
@@ -8,7 +8,7 @@
             </colgroup>
             <tbody>
                 <tr>
-                    <td class="main-option"><div>-- {{__ "edit choices"}} --<span class="icon-down"></span></div></td>
+                    <td class="main-option" dir="{{dir}}"><div>-- {{__ "edit choices"}} --<span class="icon-down"></span></div></td>
```

Comment:
May I suggest aligning the select arrow as well?
![image](https://user-images.githubusercontent.com/11692751/157849038-09092bd3-e8ca-4aac-a447-dbdd3780daf3.png)

Reply:
Does it show on the left in your local?  🤔 
In mine it shows  on the right
![image](https://user-images.githubusercontent.com/60346520/157856335-eb2bc9a3-1882-4558-bd15-d5b316c332f7.png)

Reply:
No, it shows on the right side, I just flip to show what I mean. Sorry for the confusion.